### PR TITLE
Genome-wide alternative splicing training v.2.0

### PR DESCRIPTION
**Pull request by gallardoalba**
This version allows to include new identified isoforms in the analysis. Main changes are:

- Replaced FASTQ files by sub-sampled versions (planned for previous version but it was not included by mistake)

- Includes rnaQUAST and GFFCompare QC step.

-  The former version used StingTie in reference-only mode. This update allows to [quantify also new identified isoforms](https://github.com/galaxyproject/training-material/pull/4178/files#diff-c7740d45d334b508b6af52ca67879d1492b60678ded7029b0eaa21a8541e50b7R448). 

- IsoformSwitchAnalyzeR bug fixes. Now the [violin plot shows all variants](https://github.com/galaxyproject/training-material/pull/4178/files#diff-a7c19dfc40c1b2c2971130df3747f45a6a14a4f9e000fa88af88b3c1eaae555e). Volcanoplot and scatterplot correctly generated and [included](https://github.com/galaxyproject/training-material/blob/c3cda5580e68c7fa4d9dcd3b90c78fea890ac0c4/topics/transcriptomics/images/differential_isoform/isoformSwitchAnalyzer_volcanoplot.png).

- IsoformSwitchAnalyeR settings: [short aminoacid sequences and isoforms with new STOP codons](https://github.com/galaxyproject/training-material/pull/4178/files#diff-c7740d45d334b508b6af52ca67879d1492b60678ded7029b0eaa21a8541e50b7R647) included.

- This version doesn't require to provide both transcriptome, protein coding sequences and lncRNA sequences as input. It is generated during the training by using the GTF generated by [StringTie and GFFread](https://github.com/galaxyproject/training-material/pull/4178/files#diff-c7740d45d334b508b6af52ca67879d1492b60678ded7029b0eaa21a8541e50b7R534).

- STAR step has been modified in order to include new identified splicing sites, according[ STAR developer recommendations](https://groups.google.com/g/rna-star/c/Cpsf-_rLK9I). In addition, it makes use of the parameters recommended by the [ENCODE Project pipeline](https://github.com/ENCODE-DCC/rna-seq-pipeline/blob/dev/src/align.py).

- Include plots with [additional information](https://github.com/galaxyproject/training-material/pull/4178/files#diff-589fe4da6e1dbd473e4e0eaa5b24373fb4d73e8f830258d82af9696f68fa22b0).

- pyGenomeTrack uses the splicing junction annotation[ generated by StringTie](https://github.com/galaxyproject/training-material/pull/4178/files#diff-99eca97f5c256615d6b5af5677344ceb285456b9323233fcb09df0c27c5ffcad). It allows to visualize new isoforms.

**Training v.1.0**

workflow-472-b79e689d3191b76cb75b7dc136c24c5b6265d22e-diagram

**Training v.2.0**

workflow-501-cc97fe61c6cd580f0bd6fce7cd4c04ba54ba5e69-diagram

### CODE REVIEW

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 1]
```diff
@@ -418,30 +456,60 @@ After evaluating the quality of the RNA-seq data, we can start with the transcri
...
+Despite in this training we make use of RNA STAR as mapping tool, it is possible to use different splice-aware aligner such as HISAT2. Independently of the tool, each record with a spliced alignment should have the XS tag in the SAM/BAM file, which indicates the genomic strand from which the RNA that produced the read originated.
+
+This step could be divided two substeps: transcriptome assembly and quantification isoform (figure. X). The first step will allow us to obtain a description of the transcriptome, including coordinates of whole transcriipts and exon localization. This information will be used for improving the quantification in the second step.
+
+> ![Figure 10. StringTie workflow](../../images/differential_isoform/stringtie_algorithm.png "StringTie worklow. Yellow area represent the substep corresponding to transcriptome assembly; this substep generates an annotaiton corresponding to the set of non-redundant transcripts observed in any of the RNA-Seq samples. The blue region corresponds to the quantification step, which makes use of the previously generated. Adapted from Perea et al. 2016.")
```
Comment (gallardoalba) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:481]:
```suggestion
![Figure 10. StringTie workflow](../../images/differential_isoform/stringtie_algorithm.png "StringTie worklow. Yellow area represent the substep corresponding to transcriptome assembly; this substep generates an annotaiton corresponding to the set of non-redundant transcripts observed in any of the RNA-Seq samples. The blue region corresponds to the quantification step, which makes use of the previously generated. Adapted from Perea et al. 2016.")
```

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 2]
```diff
@@ -744,20 +834,33 @@ We will try to confirm this {IS} by visualising the coverage on this gene and mo
 >        - {% icon param-repeat %} *"Insert Include tracks in your plot"*
 >            - *"Choose style of the track"*: `Gene track / Bed track`
 >                - *"Plot title"*: `Genes`
->                - {% icon param-file %} *"Track file(s) bed or gtf format"*: Select `gencode.v43.annotation.gtf.gz`
+>                - {% icon param-file %} *"Track file(s) bed or gtf format"*: Select `StringTie annotation`
```
Comment (gallardoalba) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:837]: Do you think this is fine @lldelisle? By using the annotation generated by StringTie, it allows to visualize the new identified isoforms.

Comment (gallardoalba) [reply topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:837]: Here you can see the differences https://github.com/galaxyproject/training-material/pull/4178/files#diff-99eca97f5c256615d6b5af5677344ceb285456b9323233fcb09df0c27c5ffcad

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 3]
```diff
@@ -72,18 +72,18 @@ The first step of our analysis consists of retrieving the RNA-seq datasets from
...
->      SRR9050438_Health	{{ page.zenodo_link }}/files/SRR9050438_reverse.fastqsanger.gz	fastqsanger.gz	2
->      SRR9050439_Health	{{ page.zenodo_link }}/files/SRR9050439_forward.fastqsanger.gz	fastqsanger.gz	1
->      SRR9050439_Health	{{ page.zenodo_link }}/files/SRR9050439_reverse.fastqsanger.gz	fastqsanger.gz	2
->      SRR9050440_Cancer	{{ page.zenodo_link }}/files/SRR9050440_forward.fastqsanger.gz	fastqsanger.gz	1
->      SRR9050440_Cancer	{{ page.zenodo_link }}/files/SRR9050440_reverse.fastqsanger.gz	fastqsanger.gz	2
->      SRR9050441_Cancer	{{ page.zenodo_link }}/files/SRR9050441_forward.fastqsanger.gz	fastqsanger.gz	1
->      SRR9050441_Cancer	{{ page.zenodo_link }}/files/SRR9050441_reverse.fastqsanger.gz	fastqsanger.gz	2
->      SRR9050442_Cancer	{{ page.zenodo_link }}/files/SRR9050442_forward.fastqsanger.gz	fastqsanger.gz	1
->      SRR9050442_Cancer	{{ page.zenodo_link }}/files/SRR9050442_reverse.fastqsanger.gz	fastqsanger.gz	2
+>      SRR9050437_Health	{{ page.zenodo_link }}/files/SRR9050437F_health.fastq.gz	fastqsanger.gz	1
```
[the remark this answers is not in the document]
Comment (gallardoalba) [reply topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:75]: The files were available in Zenodo, but I forgot to modify the links :(

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 4]
```diff
@@ -418,30 +456,60 @@ After evaluating the quality of the RNA-seq data, we can start with the transcri
...
+>
+> Secondly, it implements a pruning algorithm that reduces the size of the splicing graph to a more realistic size by removing the edges in the graph starting from the edge least supported by reads to the most supported edge, until the number of nodes in the splicing graph falls under a given threshold.
 >
 {: .details}
 
-Despite in this training we make use of RNA STAR as mapping tool, it is possible to use different splice-aware aligner such as HISAT2. Independently of the tool, each record with a spliced alignment should have the XS tag in the SAM/BAM file, which indicates the genomic strand from which the RNA that produced the read originated .
+Despite in this training we make use of RNA STAR as mapping tool, it is possible to use different splice-aware aligner such as HISAT2. Independently of the tool, each record with a spliced alignment should have the XS tag in the SAM/BAM file, which indicates the genomic strand from which the RNA that produced the read originated.
```
Comment (pavanvidem) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:477]:
```suggestion
Despite in this training we make use of **RNA STAR** as mapping tool, it is possible to use different splice-aware aligner such as **HISAT2**. Independently of the tool, each record with a spliced alignment should have the XS tag in the SAM/BAM file, which indicates the genomic strand from which the RNA that produced the read originated.
```

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 5]
```diff
@@ -418,30 +456,60 @@ After evaluating the quality of the RNA-seq data, we can start with the transcri
...
 
-> <hands-on-title> Task description </hands-on-title>
+This step could be divided two substeps: transcriptome assembly and quantification isoform (figure. X). The first step will allow us to obtain a description of the transcriptome, including coordinates of whole transcriipts and exon localization. This information will be used for improving the quantification in the second step.
```
Comment (pavanvidem) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:479]:
```suggestion
This step could be divided into two substeps: transcriptome assembly and quantification isoform (fig. 10). The first step will allow us to obtain a description of the transcriptome, including coordinates of whole transcripts and exon localization. This information will be used for improving the quantification in the second step.
```

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 6]
```diff
@@ -418,30 +456,60 @@ After evaluating the quality of the RNA-seq data, we can start with the transcri
...
+
+![Figure 10. StringTie workflow](../../images/differential_isoform/stringtie_pipeline.png "StringTie worklow. Yellow area represent the substep corresponding to transcriptome assembly; this substep generates an annotaiton corresponding to the set of non-redundant transcripts observed in any of the RNA-Seq samples. The blue region corresponds to the quantification step, which makes use of the previously generated. Adapted from Perea et al. 2016.")
```
Comment (pavanvidem) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:481]:
```suggestion
![Figure 10. StringTie workflow](../../images/differential_isoform/stringtie_pipeline.png "StringTie worklow. The yellow area represents the substep corresponding to transcriptome assembly; this substep generates an annotation corresponding to the set of non-redundant transcripts observed in any of the RNA-Seq samples. The blue region corresponds to the quantification step, which makes use of the previously generated. Adapted from Perea et al. 2016.")
```

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 7]
```diff
@@ -257,16 +256,56 @@ The choice of **RNA STAR** as mapper is also determined by the sequencing techno
...
+>    - *"Single-end or paired-end reads"*: `Paired-end (as collection)`
+>        - {% icon param-collection %} *"RNA-Seq FASTQ/FASTA paired reads"*: `Trimmed samples` (output of **fastp** {% icon tool %})
+>    - *"Custom or built-in reference genome"*: `Use reference genome from history and create temporary index`
+>        - {% icon param-file %} *"Select a reference genome"*: `GRCh38.p13.genome.fa.gz`
+>        - *"Build index with or without known splice junctions annotation"*: `build index with gene-model`
+>            - {% icon param-file %} *"Gene model (gff3,gtf) file for splice junctions"*: `gencode.v43.annotation.gtf.gz`
+>
+{: .hands_on}
+
+This first **RNA STAR** run generates three collections: logs, alignments in BAM format, and a collection of BED files with informtion about splice junction coordinates. Now we will use  process the collection of BED files with the objective of filtering non-canonical (column 5 > 0), unannotated (column 6 == 0) junction sites and junctions supported by too few reads (column 7 > 2).
```
Comment (pavanvidem) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:271]:
```suggestion
This first **RNA STAR** run generates three collections: logs, alignments in BAM format, and a collection of BED files with information about splice junction coordinates. Now we will process the collection of BED files with the objective of filtering non-canonical (column 5 > 0), unannotated (column 6 == 0) junction sites and junctions supported by too few reads (column 7 > 2).
```

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 8]
```diff
@@ -257,16 +256,56 @@ The choice of **RNA STAR** as mapper is also determined by the sequencing techno
...
+>   - "*On column*": `Column 1`
+>
+> 5. {% tool [Unique lines](sort1) %} assuming sorted input file with the following parameters:
+>   - {% icon param-file %} *"File to scan for unique values"*: output of **Sort** {% icon tool %}
+> 
+> 6. Rename the output as `Splicing database`
+>
+{: .hands_on}
+
+Finally we will re-run **RNA STAR** in order to integrate the information about the new splicing events in the analysis.
```
Comment (pavanvidem) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:297]:
```suggestion
Finally, we will re-run **RNA STAR** in order to integrate the information about the new splicing events in the analysis.
```

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 9]
```diff
@@ -418,30 +456,60 @@ After evaluating the quality of the RNA-seq data, we can start with the transcri
...
+> To handle the high error rates in the long reads, **StringTie** implements two techniques:
+>
+> First, it correct potentially wrong splice sites by checking all the splice sites present in the alignment of a read with a high-error alignment rate. If a splice site is not supported by any low-error alignment reads, then it tries to find a nearby splice site (within 10 bp, by default) that is supported by the most alignments among all nearby splice sites.
```
Comment (pavanvidem) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:471]:
```suggestion
> First, it corrects potentially wrong splice sites by checking all the splice sites present in the alignment of a read with a high-error alignment rate. If a splice site is not supported by any low-error alignment reads, then it tries to find a nearby splice site (within 10 bp, by default) that is supported by the most alignments among all nearby splice sites. 
```

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 10]
```diff
@@ -465,6 +533,19 @@ Despite in this training we make use of RNA STAR as mapping tool, it is possible
 >
 {: .details}
 
+Before starting with the analysis of the isoforms, we will extract the transcript sequences from the annotation generated by **Stringtie merge**. This output will be used in the next section.
```
Comment (pavanvidem) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:536]:
```suggestion
Before starting with the analysis of the isoforms, we will extract the transcript sequences from the annotation generated by **StringTie merge**. This output will be used in the next section.
```

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 11]
```diff
@@ -539,22 +620,24 @@ The first step of the IsoformSwitchAnalyzeR pipeline is to import the required d
...
 >
 {: .hands_on}
 
 It generates a **switchAnalyzeRlist** object that contains all relevant information about the isoforms involved in isoform swtches, such as each comparison of an isoform between conditions.
 
 
-
 ## Pre-processing step
 
 Once the datasets have been imported into a RData file, we can start with the pre-processing step. In order to enhace the reliability of the downstream analysis, it is important to remove the non-informative genes/isoforms (e.g. single isoform genes and non-expressed isoforms).
```
Comment (pavanvidem) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:636]:
```suggestion
Once the datasets have been imported into a RData file, we can start with the pre-processing step. In order to enhance the reliability of the downstream analysis, it is important to remove the non-informative genes/isoforms (e.g. single isoform genes and non-expressed isoforms).
```

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 12]
```diff
@@ -539,22 +620,24 @@ The first step of the IsoformSwitchAnalyzeR pipeline is to import the required d
...
 
 After the pre-processing, **IsoformSwitchAnalyzieR** performs the differential isoform usage analysis by using **DESXSeq**, which despite originally designed for testing differential exon usage, it has demonstrated to perform exceptionally well for differential isoform usage.**DEXSeq** uses generalized linear models to assess the significance of observed differences in isoform usage values between conditions, taking into account the biological variation between replicates ({% cite dexseq %}).
```
Comment (pavanvidem) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:638]:
```suggestion
After the pre-processing, **IsoformSwitchAnalyzieR** performs the differential isoform usage analysis by using **DESXSeq**, which despite originally designed for testing differential exon usage, it has been demonstrated to perform exceptionally well for differential isoform usage.**DEXSeq** uses generalized linear models to assess the significance of observed differences in isoform usage values between conditions, taking into account the biological variation between replicates ({% cite dexseq %}).
```

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 13]
```diff
@@ -724,7 +810,11 @@ Let's have a look at the generated plot (fig. 13).
 
 ![Figure 13. RGMB gene isoform expression profile plot](../../images/differential_isoform/isoformSwitchAnalyzer_gene.png "RGMB isoform expression profile plot. The plot integrates isoform structures along with the annotations, gene and isoform expression and isoform usage including the result of the isoform switch test.")
 
-In that case, we can appreciate that despite differences in overall gene expression is not statistically significant between health and cancerous tissues, there exists statistically significant {IS}: the isoform [ENST00000308234.11](https://www.uniprot.org/uniprotkb/J3KNF6/entry), which encodes the 478 aminoacid **Repulsive guidance molecule BMP co-receptor b protein** is repressed in cancer; on the other hand, the isoform [ENST00000513185.3](https://www.uniprot.org/uniprotkb/Q6NW40/entry), which encodes the 437 aminoanid **Repulsive guidance molecule B** is induced.
+In that case, we can appreciate that despite differences in overall gene expression is not statistically significant between health and cancerous tissues, there exists slarge differences between isoforms (despite not statistically significant due to reduced sample size):
```
Comment (pavanvidem) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:813]:
```suggestion
In that case, we can appreciate that despite differences in overall gene expression is not statistically significant between healthy and cancerous tissues, there exist large differences between isoforms (despite not statistically significant due to reduced sample size): 
```

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 14]
```diff
@@ -826,18 +930,11 @@ In this section we will assess whether there are differences with respect to the
...
-> - A3: Alternative 3’end acceptor site. Compared to the hypothetical pre-RNA an alternative 3’end acceptor site was used. Since it is compared to the pre-RNA, the donor site used is per definition more downstream than the pre-RNA (the downstream exon is shorter).
-> - ATSS: Alternative Transcription Start Sites. Compared to the hypothetical pre-RNA an alternative transcription start sites was used. Since it is compared to the pre-RNA, the ATSS site used is per definition more downstream the the pre-RNA.
-> - ATTS: Alternative Transcription Termination Sites. Compared to the hypothetical pre-RNA an alternative transcription Termination sites was used. Since it is compared to the pre-RNA, the ATTS site used is per definition more upstream than the pre-RNA.
+> Mutually exclusive exon is a secial case were two isoforms form the same gene contains two mutually exclusive exons and which are not found in any of the other isoforms from that gene.
```
Comment (pavanvidem) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:937]:
```suggestion
> Mutually exclusive exon is a special case where two isoforms from the same gene contain two mutually exclusive exons which are not found in any of the other isoforms from that gene.
```

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 15]
```diff
@@ -849,6 +946,9 @@ From the figure 17 , it can be hypothesised that some of the {AS} events are not
 
 ![Figure 18. Enrichment/depletion in isoform switch with consequences statistical analysis](../../images/differential_isoform/isoformSwitchAnalyzer_splicing_event.png "Comparison of differential splicing events. The fraction (and 95% confidence inter-val) of isoform switches (x-axis) resulting in gain of a specific alternative splice event (indicated by y axis) in the switch from health to cancer. Dashed line indicate no enrichment/depletion. Color indicate if FDR < 0.05 (red) or not (black).")
```
Comment (pavanvidem) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:947]:
```suggestion
![Figure 18. Enrichment/depletion in isoform switch with consequences statistical analysis](../../images/differential_isoform/isoformSwitchAnalyzer_splicing_event.png "Comparison of differential splicing events. The fraction (and 95% confidence inter-val) of isoform switches (x-axis) resulting in gain of a specific alternative splice event (indicated by y axis) in the switch from health to cancer. The dashed line indicates no enrichment/depletion and the color indicates if FDR < 0.05 (red) or not (black).")
```

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 16]
```diff
@@ -849,6 +946,9 @@ From the figure 17 , it can be hypothesised that some of the {AS} events are not
...
 
+
+<!-- Needs to be updated!! -->
+
 According with the results (fig. 18), there are not statistically significant differences in specific splicing type events between both experimental conditions. However, this result is affected by the fact that we are using only a fraction of the total data (remember that we subsampled the origital datasets in order to speed up the analysis).
```
Comment (pavanvidem) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:952]:
```suggestion
According to the results (fig. 18), there are no statistically significant differences in specific splicing type events between both experimental conditions. However, this result is affected by the fact that we are using only a fraction of the total data (remember that we subsampled the original datasets in order to speed up the analysis).
```

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 17]
```diff
@@ -999,4 +1113,6 @@ Once we have imported the workflow, we can run the pipeline on the [original dat
 
 # Conclusion
 
-Despite the large amount of RNA-seq data and computational methods available, isoform-based expression analysis is rare. Here we present a pipeline for  the statistical identification and analysis of splicing event and {IS} events with predicted functional consequences.
+Despite the large amount of RNA-seq data and computational methods available, isoform-based expression analysis is rare. Here we present a pipeline (fig. xx) for performing genome-wide alternative splicing analysis. 
+
+![Figure xxx. Genome-wide alternative splicing pipeline scheme](../../images/differential_isoform/full_workflow.png "Genome-wide isoform switching pipeline scheme. The current version (v.3) includes 39 steps.")
```
Comment (pavanvidem) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:1118]:
```suggestion
![Figure xxx. Genome-wide alternative splicing pipeline scheme](../../images/differential_isoform/full_workflow.png "Genome-wide isoform switching pipeline scheme. The current version (v.2) includes 42 steps.")
```

**Comment by gallardoalba** [comment]
Hi @lldelisle, I almost finished the update. Do you think you could have a look at the changes? Thanks!

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 18]
```diff
@@ -245,28 +239,142 @@ In that section makes use of three main tools: **RNA STAR**, considered a state-
 >
 {: .details}
 
+The choice of **RNA STAR** as mapper is also determined by the sequencing technology; it has been demonstrated adequate for short-read sequencing data, but when using long-read data, such as PacBio or ONT reads, it is recommended to use **GMAP** as alignment tool ({% cite Krianovi2017 %}).
```
Comment (gallardoalba) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:242]:
```suggestion
The choice of **RNA STAR** as mapper is also determined by the sequencing technology; it has been demonstrated adequate for short-read sequencing data, but when using long-read data, such as PacBio or ONT reads, it is recommended to use **Minimap2** as alignment tool.
```

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 19]
```diff
@@ -384,66 +490,115 @@ Now, let's evaluate the results generated by the **Gene Body Coverage** module.
...
+>       - From *"Select fasta outputs"*: `fasta file with spliced exons for each GFF transcript`
+>
+> 4. Rename the collection as `Assembled transcripts sequences`
+>
+> 5. {% tool [StringTie merge](toolshed.g2.bx.psu.edu/repos/iuc/stringtie/stringtie_merge/2.2.1+galaxy1) with the following parameters:
+>    - {% icon param-collection %} *"Transcripts"*: `Assembled transcripts` (output of **StringTie**)
+>    - {% icon param-file %} *"Reference annotation to include in the merging"*: `GRCh38.p13.chrom5.gtf`
+>
+> 6. Rename the output as `Reference transcriptome annotation`
+{: .hands_on}
```
Comment (gallardoalba) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:584]:
```suggestion
> 6. Rename the output as `Reference transcriptome annotation`
>
{: .hands_on}
```

**Review by bgruening — APPROVED** [review]
Thats an awesome update! I guess some small updates will come in the next weeks and feedback from @lldelisle would be super valuable. For now lets get this in - its great!

**Pull request merged**